keyboards/kbdfans/tofu96/tofu96.c

```diff
@@ -0,0 +1,26 @@
+#include "tofu96.h"
+
+void matrix_init_kb(void) {
+    // put your keyboard start-up code here
+    // runs once when the firmware starts up
+
+    matrix_init_user();
+    led_init_ports();
+}
+
+void led_init_ports(void) {
+    setPinOutput(B2);
+    writePinHigh(B2);
+    setPinOutput(B0);
+    writePinHigh(B0);
+	setPinOutput(B1);
+    writePinHigh(B1);
+}
```

Comment:
You shouldn't need to set the pins high, here.  IIRC, the led_update function is called at the end of each loop, so, it should be automatically set